tests/acceptance_scenarios/layers_cases.py

```diff
@@ -47,8 +47,6 @@ class LayersCases(CasesProtocol):
 
     # Only cases that should be 'realistic'.
     cases = [
-        pytest.param(with_clay_and_grass.layers_dict, id="With grass and clay layer"),
-        pytest.param(
-            with_acceptance_criteria.layers_dict, id="With acceptance criteria"
-        ),
+        pytest.param(with_clay_and_grass.layers_dict, id="With grass and clay"),
+        pytest.param(with_acc_crit.layers_dict, id="With acc crit"),
```

Comment:
I do not see the benefit of renaming `acceptance criteria` to `acc crit` please revert.

Reply:
but we are going to end up having the same problem, modifying this "just here and there" will not solve the problem, if this is a recurring problem then we need to rethink how files are being generated (perhaps hashing the name). Given the current situation perhaps it's a good idea to address this problem in a separate issue.

Reply:
Reverted and created an issue https://github.com/Deltares/Koswat/issues/237